synfig-core/src/synfig/string.h

```diff
@@ -31,6 +31,7 @@
 /* === H E A D E R S ======================================================= */
 
 #include <string>
+#include <synfig/string_helper.h>
```

Comment:
Should I merge those two files? `synfig/string.h` and `synfig/string_helper.h`

Reply:
I suggest to leave it as is for now.